Makefile

```diff
@@ -21,7 +21,7 @@ nadiff: $(obj)
 
 install: nadiff
 > mkdir -p ${PREFIX}/bin
-> cp -f nadiff ${PREFIX}/bin
+> cp -f nadiff git-nadiff ${PREFIX}/bin
 > chmod 755 ${PREFIX}/bin/nadiff
 
 uninstall:
```

Comment:
Looks good but add git-nadiff to uninstall as well

Reply:
ok